whatsapp.py

```diff
@@ -189,54 +31,55 @@ def main(path,name):
     df.head()
 
     # drop "Media omitted"
-    media_df = df[df["text"]=="<Media omitted>"]
-    df = df[df["text"]!="<Media omitted>"]
```

Comment:
Now I get why I was instructed to move files in a different PR, I can see the old commit here as files changes, but not the ones in the files that were moved. Writing this just so we'll remember for next time